Source/WebKit/UIProcess/WebProcessProxy.h

```diff
@@ -691,6 +695,10 @@ class WebProcessProxy : public AuxiliaryProcessProxy, private ProcessThrottlerCl
     std::unique_ptr<WebLockRegistryProxy> m_webLockRegistry;
     std::unique_ptr<WebPermissionControllerProxy> m_webPermissionController;
     bool m_isConnectedToHardwareConsole { true };
+#if PLATFORM(MAC) && USE(RUNNINGBOARD)
+    bool m_isSuspensionEnabled;
```

Comment:
This is missing default initialization to it is undefined what the initial value will be. This should be:
`bool m_isSuspensionEnabled { false };`